snapshot/git/gitfiler/pool.go

```diff
@@ -80,6 +92,7 @@ func (p *RepoPool) loop() {
 			p.freeList = append(p.freeList, r)
```

Comment:
I would find that update helpful. Wrt to the confusing 'size', I might prefer initited or numInited.